graphql/schema/mutation.graphql

```diff
@@ -1,33 +1,33 @@
 # This file lists all of the mutations. The mutation definitions can be found in the corresponding files in the resolvers folder.
 type Mutation {
   # annotations
-  bbCreateTicket(taskId: String!, execution: Int): Boolean!
+  bbCreateTicket(taskId: String! @requireProjectAccess(permission: ANNOTATIONS, access: EDIT), execution: Int): Boolean!
```

Comment:
nit: with the directive added, is this long enough to follow the spaced-out format of the below mutations?